api/sys_capabilities.go

```diff
@@ -2,11 +2,72 @@ package api
 
 import "fmt"
 
+// CapabilitiesSelf returns the capabilities of the client token against a given path
+// For compatibility, this is only valid when specifying a single path
 func (c *Sys) CapabilitiesSelf(path string) ([]string, error) {
 	return c.Capabilities(c.c.Token(), path)
 }
 
+// CapabilitiesSelfMultiple returns the capabilities of the client token against given paths
+func (c *Sys) CapabilitiesSelfMultiple(path string) (map[string][]string, error) {
```

Comment:
This assumes that the caller always wants to use a comma separated string, but API users are then forced into a path join. Instead this should take in an interface and use `parseutil.ParseCommaStringSlice` to get out a `[]string` from either type of input.

Reply:
That makes much more sense! 👍 

Will add that soon.